CMakeLists.txt

```diff
@@ -43,7 +43,7 @@ endif()
 set_bool(is_not_msvc_clang ${is_clang} AND NOT ${is_msvc_clang})
 
 option(STANDALONE
-  "Build UnoDB not for linking with outside code. Enables extra global checks")
+  "Build UnoDB as a standalone project. Builds tests & benchmarks, enables extra global checks")
```

Comment:
Is IWYU really required for MAINTAINER mode?  (a few lines below this).  I think it is just recommended.  clang-tidy by itself seems to do just fine (or better since not IWYU suggestions are valid).  And with gcc you don't have any of it.

Do you require clang for development or is gcc also ok?  Maybe call out in the README.md the aspects of development which do critically depend on clang.

Also, call out the requirement for clang-17 for the reformatting of text in the README and how to reflow the text during development.  Not having that in place caused a huge amount of pain :-)

Reply:
IWYU addressed, putting a note re. clang-format being v 17 ATM.